docs/_includes/video.adoc

```diff
@@ -6,6 +6,14 @@ Included in:
 
 The `video` macro supports displaying videos stored relatively as well as externally.
 
+While Asciidoctor supports many popular video formats, their support in browsers is a constantly changing landscape.
+
+.A note about serving video in browsers
+Wherever possible, you should use a video hosting service like YouTube or Vimeo to serve videos in online documentation formats. These services specialize in serving video in-browser, with the lowest latency possible given hardware, software, and network capabilities of the device viewing the video.
```

Comment:
Ahh, so that's the explanation for the pattern I'm seeing in the user
guide. I thought you just decided not to put whitespace between what looked
like terse paragraphs.

Another one for #468 ;)

On Thu, 28 Jan 2016 at 04:47 Dan Allen <notifications@github.com> wrote:

> In docs/_includes/video.adoc
> <https://github.com/asciidoctor/asciidoctor.org/pull/485#discussion_r51029601>
> :
>
> > @@ -6,6 +6,14 @@ Included in:
> >
> >  The `video` macro supports displaying videos stored relatively as well as externally.
> >
> > +While Asciidoctor supports many popular video formats, their support in browsers is a constantly changing landscape.
> > +
> > +.A note about serving video in browsers
> > +Wherever possible, you should use a video hosting service like YouTube or Vimeo to serve videos in online documentation formats. These services specialize in serving video in-browser, with the lowest latency possible given hardware, software, and network capabilities of the device viewing the video.
>
> I really like the phrasing here. I have two small suggestions.
>
> Instead of "serving video in-browser", I'd prefer "streaming optimized
> video to the browser"
>
> Also, could you use the sentence-per-line convention? After "online
> documentation formats." use a newline instead of a space. It will look like
> this:
>
> ... in online documentation formats.
> These services specialize in...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor.org/pull/485/files#r51029601>.
>
-- 

Sent from Mobile.